src/arrayMethodSort.js

```diff
@@ -5,8 +5,70 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const arr = this;
+
+    // !!!bubble sort for dividing option!!!
+    if (compareFunction && arr.length > 1 && compareFunction(arr[0],
```

Comment:
```suggestion
    if (arr.length > 1)
```
instead of that logic you just add the default value for compareFunction